tests/phpunit/tests/option/option.php

```diff
@@ -716,7 +717,7 @@ static function () use ( $default_value ) {
 
 		$this->assertIsObject( $actual, 'The option was not added to the database.' );
 		$this->assertObjectHasProperty( 'option_value', $actual, 'The "option_value" property was not included.' );
-		$this->assertSame( $default_value, $actual->option_value, 'The value was not stored as an empty string.' );
+		$this->assertSame( 'new value', $actual->option_value, 'The new value was not stored in the database.' );
```

Comment:
```suggestion
		$this->assertFalse( $actual->option_value, 'The new value was not stored in the database.' );
```
See above, so we should assert for `false` here.